### PR TITLE
Add fiat to status table

### LINKED ISSUE #464: Please add new column in /status table to show USD profit or loss
Please add new column in /status table to show USD profit or loss

### PR DESCRIPTION
**Pull request by xmatthias**
## Summary
Cleanup of `/status table` and addition of fiat currency.

Closes  #464

- removes need to import DataFrame in rpc areas
- Cleanup of rpc (Decimal conversation is not needed here)
- add fiat-currency output to status_table.

### CODE REVIEW

File: freqtrade/rpc/rpc.py [hunk 1]
```diff
@@ -135,12 +134,11 @@ def _rpc_status_table(self) -> DataFrame:
                     trade.pair,
                     shorten_date(arrow.get(trade.open_date).humanize(only_distance=True)),
                     f'{trade_perc:.2f}%'
+
```
Comment (hroff-1902) [inline freqtrade/rpc/rpc.py]: remove it? (empty line)

File: freqtrade/rpc/rpc.py [hunk 2]
```diff
@@ -135,12 +134,11 @@ def _rpc_status_table(self) -> DataFrame:
...
                 ])
 
             columns = ['ID', 'Pair', 'Since', 'Profit']
-            df_statuses = DataFrame.from_records(trades_list, columns=columns)
-            df_statuses = df_statuses.set_index(columns[0])
-            return df_statuses
+            return trades_list, columns
```
Comment (hroff-1902) [inline freqtrade/rpc/rpc.py]: why have you changed it from returning the dataframe to this tuple? Due to performance or something else?

Comment (xmatthias) [reply freqtrade/rpc/rpc.py]: if you look at where this is used - then converting this to a DataFrame makes no sense.

The output of this function is then passed to tabulate - with "columns" as column headers ...
the same can be accomplished by keeping this as list - and passing the headers back - removing the overhead of creating (and uncreating) a Dataframe.

I did not measure the performance - but just thinking about it - it's avoiding a unnecessary conversion to dataframes.

File: freqtrade/rpc/rpc.py (hunk 1 above)
Comment (xmatthias) [reply freqtrade/rpc/rpc.py]: it kindof is? (not sure - i think you're looking at a previous commit?)

**Pull request merged**